server.go

```diff
@@ -117,6 +117,12 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 }
 
 func (s *Server) HandlePacket(packet []byte) {
+	if len(packet) == 0 {
```

Comment:
IIUC, we maybe should verify (before calling handle packet) that the length of the packet is > 1 to prevent `"\n"` from being passed through to here. (It wouldn't hurt anything, but seems silly since fwict we don't do a length check earlier.)